pkg/plugin/packetparser/packetparser_linux.go

```diff
@@ -207,22 +208,28 @@ func (p *packetParser) Start(ctx context.Context) error {
 		p.callbackID = ps.Subscribe(common.PubSubEndpoints, &fn)
 	}
 
-	outgoingLinks, err := utils.GetDefaultOutgoingLinks()
-	if err != nil {
-		return err
-	}
-	if len(outgoingLinks) == 0 {
-		return errNoOutgoingLinks
-	}
-	outgoingLink := outgoingLinks[0] // Take first link until multi-link support is implemented
+	if p.cfg.DataAggregationLevel == kcfg.High {
```

Comment:
Question: Would this pretty much disable packetparser? what exactly would packetparser plugin be doing at this level? feels redundant if this is what we're doing when we can just remove this from enabled plugins

Reply:
we are only disabling attach bpf program to eth0 for high level of aggregation, if pod level metrics is enabled, packetparser would still has to attach bpf programs to veth interfaces associated with the pods.